web/src/app/forms/FormField.js

```diff
@@ -124,9 +125,10 @@ export class FormField extends React.PureComponent {
       hint,
       value: mapValue(get(value, fieldName)),
     }
+
     const InputLabelProps = {
       required: required && !optionalLabels,
-      shrink: Boolean(props.value),
+      shrink: hasValue(props.value),
```

Comment:
Can we use `_.isEmpty` instead of our own?

Also `shrink: false` the not same as not specifying shrink at all: we still want it to handle shrink on focus and whatever other behavior material would follow. We may need to stick with the original implementation and only specify the shrink prop *if* the value is non-empty.

Reply:
@KatieMSB @dctalbot What do you think of having an explicit helper? That way it's easy to remove/delete once fixed in material

Something like this maybe:
```js
// shrinkWorkaround fixes a bug in material where non-empty values
// fail to trigger the label text to shrink.
//
// Usage:
// const InputProps = {
//  // ...
//  ...shrinkWorkaround(value)
// }
function shrinkWorkaround(value) {
  if (_.isEmpty(value)) return {}
  return {shrink: true}
}
```

Reply:
I think in that case we'd want shrink to be true -- since we'd be displaying the number. But yes, we'll need to handle that. We can just add it to the conditional `_.isEmpty && !_.isNumber` or something like that. Good catch